test/e2e/service.go

```diff
@@ -1190,14 +1169,23 @@ func (t *WebserverTest) DeleteService(serviceName string) error {
 
 func (t *WebserverTest) Cleanup() []error {
 	var errs []error
-
-	for podName := range t.pods {
-		podClient := t.Client.Pods(t.Namespace)
-		By("deleting pod " + podName + " in namespace " + t.Namespace)
-		err := podClient.Delete(podName, nil)
+	for rcName := range t.rcs {
+		By("stopping RC " + rcName + " in namespace " + t.Namespace)
+		// First, resize the RC to 0.
```

Comment:
Yeah, I think it's just a wrapper—that's where I got the idea for the name "stop." Check out:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/stop.go#L100